Aula04/cifra.rb

```diff
@@ -0,0 +1,23 @@
+def caesar_cipher(cifra, number)
+  vetor_alfabeto = ('a'..'z').to_a
+  vetor_cifra = cifra.chars
+
+  vetor_cifra.each_with_index do |c, i|
+    vetor_alfabeto.each_with_index do |a, j|
+      if (c == a)
```

Comment:
```suggestion
      if c == a
```
o comum na comunidade ruby é não utilizar parenteses nas condições a não ser que seja necessário